jax/numpy/lax_numpy.py

```diff
@@ -3174,6 +3174,14 @@ def unique(ar, return_index=False, return_inverse=False,
   raise NotImplementedError(
         "np.unique is not implemented for the axis argument")
 
+@_wraps(onp.union1d)
+def union1d(ar1, ar2):
+  if ar1.ndim == 0:
+    ar1 = array([ar1])
+  if ar2.ndim == 0:
+    ar2 = array([ar2])
```

Comment:
The numpy function accepts multi-dimensional inputs:
```python
>>> np.union1d(np.arange(9).reshape(3, 3), 10)                              
array([ 0,  1,  2,  3,  4,  5,  6,  7,  8, 10])
```
It seems like the easiest way to handle this, as well as the zero-dim case, would be to call ``ravel()`` on both inputs